src/app/components/toast/index.html

```diff
@@ -61,4 +61,15 @@
       demoName="Toast">
     </sky-demo-page-code>
   </sky-demo-page-example>
+	
+	<sky-demo-page-content sectionHeading="UX guidelines">
+    <div class="sky-subsection-heading">Usage</div>
+		<p>Toasts are appropriate when:</p>
+		<ul>
+			<li>The trigger is server-side, indicating that a state in the system has changed.</li>
+			<li>The content is immediately important such that the user needs to know the information regardless of their current task or context in the system.</li>
```

Comment:
"is immediately important such that" is awkward. I'd change this and tighten up the rest of the sentence to: "The information is important enough that users to notify regardless of the current task or context in the system."